Common/Properties/SharedAssemblyInfo.cs

```diff
@@ -5,8 +5,8 @@
                                "Allows strategy research, backtesting and live trading with Equities, FX, CFD, Crypto, Options and Futures Markets.")]
 [assembly: AssemblyCopyright("QuantConnect™ 2018. All Rights Reserved")]
 [assembly: AssemblyCompany("QuantConnect Corporation")]
-[assembly: AssemblyVersion("2.4.0.1")]
+[assembly: AssemblyVersion("2.4")]
```

Comment:
Just curious, is `AssemblyVersion("2.4")` same as   `AssemblyVersion("2.4.*")` ?
I thought you had to specify all four integers _or_ use the `*` character for autonumbering.